src/output/plugins/HaikuOutputPlugin.cxx

```diff
@@ -395,6 +397,9 @@ HaikuOutput::SendTag(const Tag &tag)
 			break;
 		case TAG_GENRE:
 		case TAG_DATE:
+#if LIBMPDCLIENT_CHECK_VERSION(2,12,0)
```

Comment:
... and this condition is unnecessary, because this has nothing to do with libmpdclient.